packages/nimble-components/src/tree-item/styles.ts

```diff
@@ -120,7 +121,7 @@ export const styles = css`
     .expand-collapse-button svg {
         width: ${iconSize};
         height: ${iconSize};
-        transition: transform 0.2s ease-in;
+        transition: transform ${mediumDelay} ease-in;
```

Comment:
To keep the tree and table expand/collapse behaviors synchronized it seemed wise to update this to use the same token value. Note that there is no token for 0.2s, so I opted to use the lesser of the two values on either side (0.15s vs 0.25s).